scripts/create_index.js

```diff
@@ -38,5 +38,5 @@ client.indices.create(req, (err, res) => {
     process.exit(1);
   }
   console.log('[put mapping]', '\t', indexName, res, '\n');
-  process.exit(!!err);
```

Comment:
In the case of error we do `process.exit(1)` some lines above. This was incompatible with Node >20, because it checks that argument is number and it is not, then throws...